lnwallet/channel.go

```diff
@@ -3730,17 +3737,15 @@ func (lc *LightningChannel) validateCommitmentSanity(theirLogCounter,
 		// stuck because locally we will never put another outgoing HTLC
 		// on the channel state. The FeeBuffer should ONLY be enforced
 		// if we locally pay for the commitment transaction.
-		theirBalance, bufferAmt, err = lc.applyCommitFee(
+		theirBalance, bufferAmt, commitFee, err = lc.applyCommitFee(
 			theirBalance, commitWeight, feePerKw, NoBuffer)
 		if err != nil {
-			commitFee := feePerKw.FeeForWeight(commitWeight)
 			lc.log.Errorf("Cannot pay for the CommitmentFee "+
 				"of the ChannelState: theirBalance is "+
 				"negative after applying the fee: "+
 				"theiBalance=%v, commitFee=%v, feeBuffer=%v "+
 				"(type=%v) remote_chan_initiator",
-				int64(theirBalance), commitFee, bufferAmt,
-				buffer)
+				theirBalance, commitFee, bufferAmt, buffer)
```

Comment:
nit: could also fix the pre-existing typo in the message (s/theiBalance/theirBalance).

Reply:
ah didn't notice before - now fixed